server/drivers/hd44780-4bit.c

```diff
@@ -101,6 +101,10 @@ unsigned char lcdstat_HD44780_readkeypad(PrivateData *p, unsigned int YData);
 
 static const unsigned char EnMask[] = { EN1, EN2, EN3, STRB, LF, INIT, SEL };
 
+/*function set buffer to store during init*/
+static unsigned char bufferFUNCSET = 0;
+
```

Comment:
The LCDd driver API doesn't allow for static variables like this. You need to add this to the driver private data instead.